EFImageViewZoom.swift

```diff
@@ -30,7 +30,7 @@ private extension UIScrollView {
 public class EFImageViewZoom: UIScrollView {
     
     public weak var _delegate: EFImageViewZoomDelegate?
-    fileprivate var imageView: UIImageView!
+    fileprivate (set) public var imageView: UIImageView!
```

Comment:
this allows you to do something like 
... var view: EFImageViewZoom ...
view.imageView.loadImage(from: URL)